molecule/null-cr-values-test/kiali-cr.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   additional_display_details: null
   installation_tag: null
   istio_component_namespaces: null
-  istio_namespace: {{ istio.control_plane_namespace }}
+  istio_namespace: { { istio.control_plane_namespace } }
```

Comment:
This file got all screwed up. First, this is incorrect Ansible and won't run. No spaces between the two braces are allowed. Looks like this happened all throughout this file.